signer/tests/integration/postgres.rs

```diff
@@ -1424,14 +1424,23 @@ async fn get_swept_deposit_requests_returns_swept_deposit_requests() {
     // can rely on that fact.
     setup.store_deposit_tx(&db).await;
 
+    // The request needs to be added to the database. This stores
+    // `setup.deposit_request` into the database.
+    setup.store_deposit_request(&db).await;
+
+    // TODO: Create the initial transaction sweep package without any
+    // withdrawals and have a separate method for creating that sweep (since
+    // it's not realistic to have the withdrawal in the same sweep as the
+    // deposit). Then we wouldn't have to do this.
+    setup.store_withdrawal_request(&db).await;
```

Comment:
Not sure if this is relevant to this comment but withdrawal request stored here is just random. It's not tied to the deposit.

Reply:
Yeah the problem is that the setup helper creates a sweep tx with the withdrawal in it (which is what i meant by the TODO, that we should leave the withdrawal out of the first sweep tx), so the FK breaks if we haven't stored the withdrawal request.

Reply:
The comment is more related that we should, in the sweep setup helper, not include the withdrawal in the first sweep transaction and instead make a new sweep transaction when we're actually processing the withdrawal. Right now we have to insert the withdrawal request because the sweep tx will include it and otherwise break the FK.

Reply:
Ahh I see. Makes sense.